blocks/contrast-checker/test/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+import tinycolor from 'tinycolor2';
+
+/**
+ * Internal dependencies
+ */
+import ContrastChecker from '../';
+
+describe( 'ContrastChecker', () => {
+	const backgroundColor = '#ffffff';
+	const textColor = '#000000';
+	const isLargeText = true;
+	const fallbackBackgroundColor = '#fff';
+	const fallbackTextColor = '#000';
+	const sameShade = '#666';
+
+	const wrapper = shallow(
+		<ContrastChecker
+			backgroundColor={ backgroundColor }
+			textColor={ textColor }
+			isLargeText={ isLargeText }
+			fallbackBackgroundColor={ fallbackBackgroundColor }
+			fallbackTextColor={ fallbackTextColor } />
+	);
+
+	test( 'should render null when no colors are provided', () => {
+		expect( shallow( <ContrastChecker /> ).html() ).toBeNull();
+	} );
+
+	test( 'should render null when the colors meet AA WCAG guidelines.', () => {
+		const readable = tinycolor.isReadable(
+			backgroundColor,
+			textColor,
+			{ level: 'AA', size: ( isLargeText ? 'large' : 'small' ) }
+		);
+
+		expect( readable ).toBe( true );
+		expect( wrapper.html() ).toBeNull();
+	} );
+
+	test( 'should render component when the colors do not meet AA WCAG guidelines.', () => {
+		const readable = tinycolor.isReadable(
+			sameShade,
+			sameShade,
+			{ level: 'AA', size: ( isLargeText ? 'large' : 'small' ) }
+		);
+
+		const componentWrapper = shallow(
+			<ContrastChecker
+				backgroundColor={ sameShade }
+				textColor={ sameShade }
+				isLargeText={ isLargeText }
+				fallbackBackgroundColor={ fallbackBackgroundColor }
+				fallbackTextColor={ fallbackTextColor } />
```

Comment:
Given we are not testing something related to fall back colors in this test case and in test case "should render different message matching snapshot when background color has less brightness than text color and different text size.", I think it would be better to not pass fallback colors in both of this cases. And then add a test that would make sure that if both normal colors and fall colors are passed the message shown is the result of the usage of normal colors.